x/tokenomics/keeper/settle_pending_claims.go

```diff
@@ -305,15 +331,97 @@ func (k Keeper) proofRequirementForClaim(ctx sdk.Context, claim *prooftypes.Clai
 	}
 
 	logger.Info(fmt.Sprintf(
-		"claim does not require proof due to compute units (%d) being less than the threshold (%d) and random sample (%.2f) being greater than probability (%.2f)",
-		numClaimComputeUnits,
+		"claim does not require proof due to claimed amount (%s) being less than the threshold (%s) and random sample (%.2f) being greater than probability (%.2f)",
+		claimeduPOKT,
 		proofParams.GetProofRequirementThreshold(),
 		proofRequirementSampleValue,
 		proofParams.GetProofRequestProbability(),
 	))
 	return requirementReason, nil
 }
 
+// slashSupplierStake slashes the stake of a supplier slashingCount times and mints
+// the total slashing amount to the tokenomics module account.
+func (k Keeper) slashSupplierStake(
+	ctx sdk.Context,
+	supplierOperatorAddress string,
+	slashingCount uint64,
+) error {
+	logger := k.logger.With("method", "slashSupplierStake")
+
+	proofParams := k.proofKeeper.GetParams(ctx)
+	slashingPenalty := proofParams.GetProofMissingPenalty()
+
+	totalSlashingAmt := slashingPenalty.Amount.Mul(math.NewIntFromUint64(slashingCount))
+	totalSlashingCoin := sdk.NewCoin(volatile.DenomuPOKT, totalSlashingAmt)
+
+	supplierToSlash, supplierFound := k.supplierKeeper.GetSupplier(ctx, supplierOperatorAddress)
+	if !supplierFound {
+		return types.ErrTokenomicsSupplierNotFound
+	}
+
+	remainingStakeAmt := math.NewInt(0)
+	if supplierToSlash.GetStake().Amount.GT(totalSlashingAmt) {
+		remainingStakeAmt = supplierToSlash.GetStake().Amount.Sub(totalSlashingAmt)
+	}
+
+	err := k.bankKeeper.MintCoins(ctx, types.ModuleName, sdk.NewCoins(totalSlashingCoin))
+	if err != nil {
+		return err
+	}
+
+	supplierToSlash.Stake.Amount = remainingStakeAmt
+
+	logger.Warn(fmt.Sprintf(
+		"slashing supplier %s stake by %s, remaining stake: %s",
+		supplierToSlash.GetOperatorAddress(),
+		totalSlashingCoin,
+		remainingStakeAmt,
+	))
+
+	// Check if the supplier's stake is below the minimum and unstake it if necessary.
+	// TODO_BETA(@red-0ne, #612): Use minimum stake governance parameter once available.
+	// TODO_BETA(@red-0ne): Since SettlePendingClaims is not necessarily called
+	// at session end height so the unstaked supplier may not be immediately removed.
+	// Ensure that Gateways and Applications do not interact with a supplier that
+	// is below the minimum stake.
+	if remainingStakeAmt.LT(math.NewInt(1)) {
```

Comment:
move `math.NewInt(1)` to a local variable w/ a TODO there.

Let's assign it to @bryanchriswhite actually.